code/game/gamemodes/gang/gang.dm

```diff
@@ -1,7 +1,7 @@
 //gang.dm
 //Gang War Game Mode
 
-var/list/gang_name_pool = list("Clandestine", "Prima", "Zero-G", "Max", "Blasto", "Waffle", "North", "Omni", "Newton", "Cyber", "Donk", "Gene", "Gib", "Tunnel", "Diablo", "Psyke", "Osiron", "Sirius", "Sleeping Carp")
```

Comment:
If that's the consensus sure. I didn't want to delete someone's super cool snowflakey gang if they want to come back and make a serious effort at it.